cbits/win32/runProcess.c

```diff
@@ -617,7 +617,22 @@ waitForJobCompletion ( HANDLE hJob )
         return true;
       }
 
-      HANDLE pHwnd = OpenProcess(SYNCHRONIZE, TRUE, pid_list->ProcessIdList[0]);
+      HANDLE pHwnd;
+      bool found_p;
+
+      /* Find the first non-terminated handle.  */
+      for (DWORD i = 0; i < pid_list->NumberOfProcessIdsInList; i++) {
+        pHwnd = OpenProcess(SYNCHRONIZE, FALSE, pid_list->ProcessIdList[0]);
```

Comment:
shouldn't this be `pid_list->ProcessIdList[i]`?

Reply:
oops, indeed